lib/valkyrie/persistence/fedora/metadata_adapter.rb

```diff
@@ -10,40 +10,68 @@ module Valkyrie::Persistence::Fedora
   #   )
   class MetadataAdapter
     attr_reader :connection, :base_path, :schema
+
+    # Constructor
```

Comment:
I don't think `Constructor` is useful as a comment. It seems about as informative as the name of the method.